plasmapy/utils/decorators/tests/test_validators.py

```diff
@@ -204,45 +204,35 @@ def test_vq_method__get_validations(self):
             CheckValues, "_get_value_checks", return_value={}
         ) as mock_cv_get:
             vq = ValidateQuantities(x=u.cm)
-            vq.f = self.foo
-            sig = inspect.signature(self.foo)
+            vq.f = _foo
+            sig = inspect.signature(_foo)
             bound_args = sig.bind(5)
 
             assert vq._get_validations(bound_args) == {}
             assert mock_cu_get.called
             assert mock_cv_get.called
 
```

Comment:
Since we removed the test case that raise exceptions from `test_vq_method__get_validations` we need to add a new test method to handle these...

```suggestion

    @pytest.mark.parametrize(
        "func, func_args, func_kwargs, validations, _raises",
        [
            # call w/o unit validations
            (_foo, (5,), {}, {"x": {"can_be_inf": False}}, ValueError),
            #
            # 'none_shall_pass' is inconsistently doubly defined'
            (
                _foo,
                (5,),
                {},
                {"x": {"units": [u.cm, None], "none_shall_pass": False}},
                ValueError,
            ),
        ],
    )
    def test_vq_method__get_validations_raises(
        self, func, func_args, func_kwargs, validations, _raises
    ):
        sig = inspect.signature(func)
        bound_args = sig.bind(*func_args, **func_kwargs)

        vq = ValidateQuantities(**validations)
        vq.f = func  # manually set f attr needed for _get_validations

        with pytest.raises(_raises):
            vq._get_validations(bound_args)

```